base-api.Tests/V1/Domain/ExampleEntityTests.cs

```diff
@@ -0,0 +1,26 @@
+using System;
+using NUnit.Framework;
+using base_api.V1.Domain;
+
+namespace UnitTests.V1.Domain
+{
+    [TestFixture]
+    public class ExampleEntityTests
+    {
+        [Test]
+        public void ExampleEntitiesHaveABalance()
```

Comment:
Could you please update the test name to 'ExampleEntitiesHaveABalance' to 'ExampleEntitiesHaveId' just for clarity reasons in the future?